docs/basic-features/image-optimization.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Next.js supports built-in image optimization as well as 3rd party loaders for Imgix, Cloudinary, and more! Learn more here.
+---
+
+# Image Optimization
+
+<details open>
+  <summary><b>Examples</b></summary>
+  <ul>
+    <li><a href="https://github.com/vercel/next.js/tree/canary/examples/basic-image-optimization">Basic Image Optimization</a></li>
+  </ul>
+</details>
+
+Since version **10.0.0**, Next.js provides an Image component that will automatically optimize the source image based on the browser. The optimization includes resizing the image based on the browser's viewport as well as selecting the best format, such as [WebP](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types), based on the browser's [`Accept`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept) header.
```

Comment:
```suggestion
Since version **10.0.0**, Next.js provides an Image component to automatically optimize the source image. The optimization includes resizing the image based on the browser's viewport, as well as selecting the best format (such as [WebP](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types)) based on the browser's [`Accept`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept) header.
```